datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTab.tsx

```diff
@@ -20,6 +20,10 @@ const NoSchema = styled(Empty)`
     padding-top: 60px;
 `;
 
+const SchemaTableContainer = styled.div`
+    overflow: auto;
+    height: 100vh;
```

Comment:
it might be better to have this be `height: 100%;` but after checking locally I don't believe it really matters in this situation

Reply:
Doesn't this mean that we are forcing schema tab height to the height of the browser window?

Reply:
that is indeed what it's saying, but in this case it doesn't make a difference since this div can never grow higher than its parent. So in theory it's working the same as `height: 100%` but only in this situation. If there was no parent or if this was absolutely positioned it would fill the whole content and be the view height of the browser window.

All that being said I think her we actually just change it to be `100%` instead of `100vh` even though functionally there the same here (just better css practice)